utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -799,15 +800,25 @@ Describe 'Module tests' -Tag 'Module' {
                 }
 
                 if ($incorrectParameters.Count -gt 0) {
-                    $warningMessage = 'All parameters which are of type [object] or [array-of-objects] should implement a user-defined, or resource-derived type. Found incorrect items '
-                    Write-Warning ("$warningMessage`n- {0}`n" -f ($incorrectParameters -join "`n- "))
 
-                    Write-Output @{
-                        Warning = ("$warningMessage<br>- <code>{0}</code><br>" -f ($incorrectParameters -join '</code><br>- <code>'))
+                    $versionFilePath = Join-Path (Split-Path $templateFilePath) 'version.json'
+                    $hasVersionFile = Test-Path $versionFilePath
+                    if ($hasVersionFile) {
+                        $moduleVersion = [version](Get-Content $versionFilePath -Raw | ConvertFrom-Json).version
+                    }
+                    if ((Test-Path $versionFilePath) -and $moduleVersion -ge [version]'1.0') {
```

Comment:
Why retesting if we got a var
```suggestion
                    if ($hasVersionFile -and $moduleVersion -ge [version]'1.0') {
```

Reply:
maybe I'm missing something. In this care the var is created in this test (2 lines above)